docs/LSPApi.md

```diff
@@ -14,6 +14,7 @@ The [LSPClientFeatures](https://github.com/redhat-developer/lsp4ij/blob/main/src
 - [LSP documentSymbol feature](#lsp-documentSymbol-feature)

```

Comment:
I recommend that you disable whitespace differences because I ran several of the existing files through the Java formatter against the project's code style and it resulted in minor whitespace changes.